backend/test/integration/pipeline_version_api_test.go

```diff
@@ -291,24 +291,17 @@ func (s *PipelineVersionApiTest) TestPipelineVersionAPI() {
 	argumentYAMLPipelineVersion, err = s.pipelineUploadClient.UploadPipelineVersion("../resources/arguments-parameters.yaml", pipelineVersionParams)
 	assert.Nil(t, err)
 	assert.Equal(t, "arguments-parameters.yaml", argumentYAMLPipelineVersion.Name)
-
-	versions, _, _, err := s.pipelineClient.ListPipelineVersions(&params.ListPipelineVersionsParams{ResourceKeyID: &pipelineID})
+	sortBy := "created_at"
```

Comment:
When sorting by created_at you should add a
```
time.Sleep(1 * time.Second)
```
before the second upload statement

because KFP db saves `created_at` at precision of 1 second